From f1c11281dc67e5c61a5d0c66cfdb6d6aca9c1eba Mon Sep 17 00:00:00 2001
From: Tyson Gern <tgern@pivotal.io>
Date: Fri, 21 Jul 2017 09:26:37 -0600
Subject: [PATCH 01/16] Initial commit

---
 .gitignore | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..4fa332f
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,5 @@
+build
+.gradle
+.idea
+*.iml
+ci/variables.yml
-- 
2.12.3

